ERCS/eip-send-tx-conditional.md

```diff
@@ -0,0 +1,158 @@
+---
+eip: send-tx-cond
```

Comment:
```suggestion
eip: 7796
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.